src/components/CreatePool.vue

```diff
@@ -0,0 +1,166 @@
+<template lang="">
```

Comment:
what does `lang=""` do?

It's causing my IDE to throw errors lol, changing it just to <template> fixes my errors.

Reply:
I've been messing with my IDE extensions lately, causing my templates to get messy. sorry